docs/docs/spark-procedures.md

```diff
@@ -972,4 +972,100 @@ CALL catalog_name.system.compute_table_stats(table => 'my_table', snapshot_id =>
 Collect statistics of the snapshot with id `snap1` of table `my_table` for columns `col1` and `col2`
 ```sql
 CALL catalog_name.system.compute_table_stats(table => 'my_table', snapshot_id => 'snap1', columns => array('col1', 'col2'));
-```
\ No newline at end of file
+```
+
+## Table Replication
+
+The `rewrite-table-path` procedure prepares an Iceberg table for moving or copying to another location.
```

Comment:
How can we achieve moving with this procedure?

Reply:
Hey @manuzhang, there's a section later highlights 3rd party tool such as distcp can be used to copy or move the files from source to target.

Reply:
I don't think distcp can "move" the files, right?

Reply:
For some of our use cases where require migrating many iceberg tables from one file system to another, we built tools and systems around this rewrite-table-path spark actions for 3 step process of rewrite, copy and reregister  to complete migration. Technically speaking move can be done via copy from source to target first and later delete at source but I think there's much more involved and we dont want to prescribe the solution here